lib/albacore/project.rb

```diff
@@ -210,6 +210,33 @@ def to_s
       path
     end
 
+    # Get Property folder path
+    def property_folder_path
+      read_property('AppDesignerFolder')
+    end
+
+    # Return path to properties folder
+    def properties_path
+      File.join @proj_path_base, property_folder_path
+    end
+
+    # Reads assembly version information
+    # @return string
+    def read_assembly_version
+      info= File.read(assembly_info)
+      v   = info.each_line
+                .select { |l| !(l.start_with?('//')||l.start_with?('/*')) && l.include?('AssemblyVersion') }.first
+      reg = /"(.*?)"/
+      reg.match(v).captures.first
+
+    end
+
+    # Get path to assemblyinfo.cs
+    # @return string
+    def assembly_info
```

Comment:
Let's call it something like `default_assembly_info` instead. And this wouldn't work for F# projects or C++ projects, so you should introduce a switch statement on e.g. the extension of the project file name/language. F# goes into the same folder as the project file named `AssemblyInfo.fs` and C++ I don't really care that much about ;)